tests/stress/km/stress_tests_km.cpp

```diff
@@ -1048,6 +1050,283 @@ _print_test_control_info(const test_control_info& test_control_info)
     }
 }
 
+static void
+_cleanup_tailcall_program(const fd_t& prog_map_fd)
+{
+    // Cleanup tail calls
+    for (int index = 0; index < MAX_TAIL_CALL_PROGS; index++) {
+        REQUIRE(bpf_map_update_elem(prog_map_fd, &index, &ebpf_fd_invalid, 0) == 0);
+    }
+
+    LOG_VERBOSE("{} Cleanup done for map={}", __func__, prog_map_fd);
+}
+
+static fd_t
+_setup_tailcall_program(bpf_object* object, const std::string map_name)
+{
+    REQUIRE(object != nullptr);
+
+    fd_t prog_map_fd = bpf_object__find_map_fd_by_name(object, map_name.c_str());
+    if (prog_map_fd < 0) {
+        LOG_ERROR(
+            "({}) FATAL ERROR: bpf_object__find_map_fd_by_name({}) failed. Errno:{}",
+            __func__,
+            map_name.c_str(),
+            errno);
+        REQUIRE(prog_map_fd >= 0);
+    }
+    LOG_VERBOSE("({}) Opened fd:{} for map:{}", __func__, prog_map_fd, map_name.c_str());
+
+    // Setup tail calls
+    for (int index = 0; index < MAX_TAIL_CALL_PROGS; index++) {
+        try {
+            std::string bind_program_name{"BindMonitor_Callee"};
+            bind_program_name += std::to_string(index);
+
+            // Try to get a handle to the 'BindMonitor_Calleexxindex' program.
```

Comment:
looks too close to a typo, maybe:
```suggestion
            // Try to get a handle to the 'BindMonitor_Callee<index>' program.
```